src/main/java/com/higgsup/kpi/glossary/ErrorCode.java

```diff
@@ -2,24 +2,22 @@
 
 public enum ErrorCode {
     NOT_FIND(900, "not find"),
-    NOT_FIND_USER(980,"Not find user"),
     PARAMETERS_IS_NOT_VALID(901, "parameter is not valid"),
-    PARAMETERS_ALREADY_EXIST(932,"parameters already exist"),
-    NO_LARGER_THAN(940,"no larger than"),
+    PARAMETERS_ALREADY_EXIST(932, "parameters already exist"),
+    NO_LARGER_THAN(940, "no larger than"),
     NOT_NULL(903, "parameter is not null"),
     JSON_PROCESSING_EXCEPTION(904, "json processing exception"),
     DATA_EXIST(902, "data exist"),
-    NOT_FILLING_ALL_INFORMATION(905, "Not filling all information"),
-    ALREADY_CREATED(906, "Already created"),
-    DUPLICATED_ITEM(926, "Duplicated item"),
-    DO_NOT_EXISTED_ITEM(927, "This team building activity do not existed"),
-    NOT_FIND_ITEM(928, "Item does not existed"),
-    NOT_FIND_GROUP_TYPE(929, "Item does not existed"),
-    ERROR_IO_EXCEPTION(930, "Error IOException"),
-    TEAMBUILDING_PRIZE_SCORE_CAN_NOT_NULL(931, "Team building prize score can not be null");
-
+    NOT_FILLING_ALL_INFORMATION(905, "not filling all information"),
+    ALREADY_CREATED(906, "already created"),
+    DUPLICATED_ITEM(926, "duplicated item"),
+    NOT_FIND_ITEM(928, "item does not existed"),
+    NOT_FIND_GROUP_TYPE(929, "item does not existed"),
```

Comment:
Không nên ghi item does not existed, bị trùng message với ý trên, sửa thành group type does not existed

Reply:
ok